lib/modules/datasource/nuget/__snapshots__/index.spec.ts.snap

```diff
@@ -211,6 +211,10 @@ exports[`modules/datasource/nuget/index getReleases processes real data (v2) 1`]
 
 exports[`modules/datasource/nuget/index getReleases processes real data (v3) feed is a nuget.org 1`] = `
 {
+  "datasourceProvidedReleaseNotes": "This package includes the NUnit 3 framework assembly, which is referenced by your tests. You will need
```

Comment:
The syntax higlighting here shows red outlines one some characters in the code. I think there's some kind of formatting issue here, or missing escapes for control characters?

Reply:
This is a draft PR and nowhere near functional so I've not yet reviewed any of this. A self review will follow long before this is a real PR. Draft PR is here so I can get larger guidance when it comes time. Figured I'd work out loud.

But on the topic specifically, I noticed it but it appeared to affect nothing and I couldn't get an actual error to appear, and all of the tests pass. If the CI build is approved and a linter runs maybe I'll get some specific guidance.

Reply:
Doing work in the open is fine by me!

As for the lint error, here's the relevant quote from our [local development guide](https://github.com/renovatebot/renovate/blob/main/docs/development/local-development.md):

> ## Linting and formatting
> 
> We use [Prettier](https://github.com/prettier/prettier) to format our code.
> If your code fails `pnpm test` due to a `prettier` rule then run `pnpm lint-fix` to fix it or most `eslint` errors automatically before running `pnpm test` again.
> You usually don't need to fix any Prettier errors by hand.
> 
> If you're only working on the documentation files, you can use the `pnpm doc-fix` command to format your work.